packages/features/ee/teams/pages/team-profile-view.tsx

```diff
@@ -167,6 +167,7 @@ const ProfileView = () => {
                     name: values.name,
                     slug: values.slug,
                     bio: values.bio,
+                    logo: values.logo,
```

Comment:
Teams' logo were not being updated as it wasn't counted as a change in the form before submitting